### ISSUE TITLE
RuntimeError: Error(s) in loading state_dict for DPTDepthModel

### ISSUE THREAD

**Issue by lwneal** [paulengstler/invisible-stitch#5]
As of commit d348fc, following the instructions in the README I am able to install all submodules and required packages and run the example test script. It seems to fail while attempting to load the weights for MiDaS (https://github.com/isl-org/MiDaS).

Example output:
```
$ PYTHONPATH=./utils python3 run.py   --image "examples/photo-1667788000333-4e36f948de9a.jpeg"   --prompt "a street with traditional buildings in Kyoto, Japan"   --output_path "./output.ply"   --mode "stage"
img_size [384, 512]
Using cache found in ~/.cache/torch/hub/intel-isl_MiDaS_master
Traceback (most recent call last):
  File "~/invisible-stitch/run.py", line 221, in <module>
    fire.Fire(generate_scene)
  File "~/invisible-stitch/venv/lib/python3.10/site-packages/fire/core.py", line 143, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "~/invisible-stitch/venv/lib/python3.10/site-packages/fire/core.py", line 477, in _Fire
    component, remaining_args = _CallAndUpdateTrace(
  File "~/invisible-stitch/venv/lib/python3.10/site-packages/fire/core.py", line 693, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "~/invisible-stitch/run.py", line 181, in generate_scene
    zoe_dc_model = get_zoe_dc_model(ckpt_path=hf_hub_download(repo_id="paulengstler/invisible-stitch", filename="invisible-stitch.pt")).to(device)
  File "~/invisible-stitch/utils/models.py", line 66, in get_zoe_dc_model
    return ZoeD_N(vanilla=vanilla, ckpt_path=ckpt_path, **kwargs)
  File "~/invisible-stitch/utils/models.py", line 39, in ZoeD_N
    model = build_model(zoedepth_config)
  File "~/invisible-stitch/zoedepth/models/builder.py", line 51, in build_model
    return get_version(config.version_name).build_from_config(config)
  File "~/invisible-stitch/zoedepth/models/zoedepth/zoedepth_v1.py", line 262, in build_from_config
    return ZoeDepth.build(**config)
  File "~/invisible-stitch/zoedepth/models/zoedepth/zoedepth_v1.py", line 252, in build
    core = MidasCore.build(midas_model_type=midas_model_type, use_pretrained_midas=use_pretrained_midas,
  File "~/invisible-stitch/zoedepth/models/base_models/midas.py", line 346, in build
    midas = torch.hub.load("intel-isl/MiDaS", midas_model_type,
  File "~/invisible-stitch/venv/lib/python3.10/site-packages/torch/hub.py", line 568, in load
    model = _load_local(repo_or_dir, model, *args, **kwargs)
  File "~/invisible-stitch/venv/lib/python3.10/site-packages/torch/hub.py", line 597, in _load_local
    model = entry(*args, **kwargs)
  File "~/.cache/torch/hub/intel-isl_MiDaS_master/hubconf.py", line 51, in DPT_BEiT_L_384
    model.load_state_dict(state_dict)
  File "~/invisible-stitch/venv/lib/python3.10/site-packages/torch/nn/modules/module.py", line 2189, in load_state_dict
    raise RuntimeError('Error(s) in loading state_dict for {}:\n\t{}'.format(
RuntimeError: Error(s) in loading state_dict for DPTDepthModel:
	Unexpected key(s) in state_dict: "pretrained.model.blocks.0.attn.relative_position_index", "pretrained.model.blocks.1.attn.relative_position_index", ...
```

Ubuntu 22.04.4, GTX 1080Ti, Nvidia drivers 535.171.04, Cuda 12.1

Maybe this is a known issue or a configuration problem?

**Comment by paulengstler** [comment]
Could you double-check if you have `timm==0.6.7` installed? This is related to an issue with MiDaS requiring certain version of `timm` (see https://github.com/isl-org/MiDaS/issues/245).

**Comment by lwneal** [comment]
Thank you, confirmed fixed by updating to 8156c79, installing `timm==0.6.7` and also installing `torch==2.2.0` `torchvision==0.18.0` and `xformers==0.0.24` following the instructions from https://github.com/paulengstler/invisible-stitch/issues/6

**Comment by painebenjamin** [comment]
Adding a +1 to the solution, and  wanted to note that it was _just_ the `timm` version that fixed it for me, I didn't need to change anything else.
